actions/retrieve_videos_MOD.js

```diff
@@ -0,0 +1,134 @@
+    const type = parseInt(data.storage)
+    if (type !== varType) return
+    const dataType = 'Video List'
+    return ([data.varName, dataType])
+  },
+
+  // ---------------------------------------------------------------------
+  // Action Fields
+  //
+  // These are the fields for the action. These fields are customized
+  // by creating elements with corresponding IDs in the HTML. These
+  // are also the names of the fields stored in the action's JSON data.
+  // ---------------------------------------------------------------------
+
+  fields: ['id', 'storage', 'varName'],
+
+  // ---------------------------------------------------------------------
+  // Command HTML
+  //
+  // This function returns a string containing the HTML used for
+  // editting actions.
+  //
+  // The "isEvent" parameter will be true if this action is being used
+  // for an event. Due to their nature, events lack certain information,
+  // so edit the HTML to reflect this.
+  //
+  // The "data" parameter stores constants for select elements to use.
+  // Each is an array: index 0 for commands, index 1 for events.
+  // The names are: sendTargets, members, roles, channels,
+  //                messages, servers, variables
+  // ---------------------------------------------------------------------
+
+  html: function (isEvent, data) {
+    return `
+<div style="width: 90%;">
+  YouTube Channel ID:<br>
+  <input id="id" class="round" type="text">
+</div><br>
+</div><br>
+<div style="padding-top: 8px;">
+  <div style="float: left; width: 35%;">
+    Store In:<br>
+    <select id="storage" class="round">
+      ${data.variables[1]}
+    </select>
+  </div>
+  <div id="varNameContainer" style="float: right; width: 60%;">
+    Variable Name:<br>
+    <input id="varName" class="round" type="text">
+  </div>
+</div>`
+  },
+
+  // ---------------------------------------------------------------------
+  // Action Editor Init Code
+  //
+  // When the HTML is first applied to the action editor, this code
+  // is also run. This helps add modifications or setup reactionary
+  // functions for the DOM elements.
+  // ---------------------------------------------------------------------
+
+  init: function () {},
+
+  // ---------------------------------------------------------------------
+  // Action Bot Function
+  //
+  // This is the function for the action within the Bot's Action class.
+  // Keep in mind event calls won't have access to the "msg" parameter,
+  // so be sure to provide checks for variable existance.
+  // ---------------------------------------------------------------------
+
+  action: async function (cache) {
+    const data = cache.actions[cache.index]
+    const storage = parseInt(data.storage)
+    const varName = this.evalMessage(data.varName, cache)
+    const id = this.evalMessage(data.id, cache)
+    const Mods = this.getMods()
+    const ytch = Mods.require('yt-channel-info')
+    const channelId = id
+    const sortBy = 'newest'
```

Comment:
You could actually include a field for the sorting to make the mod more versatile. The three sort modes are `newest`, `oldest`, `popular`.